src/rsz/src/SizeMove.cc

```diff
@@ -169,20 +169,6 @@ bool SizeMove::replaceCell(Instance* inst, const LibertyCell* replacement)
         || resizer_->getParasiticsSrc() == ParasiticsSrc::detailed_routing) {
       opendp_->legalCellPos(db_network_->staToDb(inst));
     }
-    if (resizer_->haveEstimatedParasitics()) {
```

Comment:
I don't see the addition of a guard to this file.  What replaces this?

Reply:
There's no need to replace this. If we are inside incremental parasitics the callback does the invalidation, if we are outside then the `invalidateParasitics()` call below was a no-op

Reply:
SizeMove::doMove has no guard either so what triggers the invalidation for this move?

Reply:
These moves are called from repairSetup which is in a guard.